painttiming.bs

```diff
@@ -62,9 +89,28 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Contentful</dfn>: An [=element=] or a [=pseudo-element=] counts as contentful when one or more of the following apply:
+* The element has a [=text node=] child, with at least one character, excluding [=document white space characters=].
+* The element is a [=replaced image=] representing a [=loaded=] image.
```

Comment:
Should replaced image link to https://www.w3.org/TR/CSS2/conform.html#replaced-element instead?

Reply:
I think the one I chose is more specific to replaced images...